### PR TITLE
social icons are enhanced 

### PR DESCRIPTION
**Pull request by Ankita297**
## What is the change?
social icons are enhanced 
## Related issue?
close: #820 

## How was it tested?
I have tested it in my local system 

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

https://user-images.githubusercontent.com/67833077/117541062-bd406e80-b02f-11eb-9c13-a384b6276935.mp4

Gurujii com - Google Chrome 08-05-2021 18_56_40

### CODE REVIEW

**Comment by Ankita297** [comment]
@Pranav016 Please review my PR .

**Comment by Pranav016** [comment]
> @Pranav016 Please review my PR .

Please don't tag unless it's very urgent or it's been more than 36hrs since PRs been made.

**Comment by Ankita297** [comment]
Sorry Sir  . I will not repeat it .

**Review by Pranav016 — CHANGES_REQUESTED** [review]
In the mobile view screenshot you added, there is a horizontal scroll bar visible which shouldn't be there.
I think it because the size of the icons is not shrinking for mobile view, even in mobile view they look big. I'd suggest that for their size instead of rem use `vw` unit and also for font size where `px` unit was used to set it.

**Comment by Ankita297** [comment]
WordNook - Google Chrome 09-05-2021 09_36_54

File: public/css/styles.css [hunk 1]
```diff
@@ -763,6 +735,90 @@ input[name='perPage'] {
...
+}
+@media only screen and (max-width: 976px) {
+.rounded-social-buttons .social-button {
+	width: 2.75rem;
+	height:2.75rem;
+	line-height: 1em;
+}
+.rounded-social-buttons .fa-twitter, .fa-facebook, .fa-linkedin, .fa-youtube, .fa-instagram {
+	   margin-top: 0.75rem;
+		font-size: 20px;
```
Comment (Pranav016) [inline public/css/styles.css:777]: size is hard coded, instead remove this media query and set a relative size to it in the first place itself.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
There is a lot of unnecessary css used like the media queries etc which is cluttering the code and that could be avoided just by using the correct units. Don't stick to `rem` for all the values, use `vw` or `vh` wherever you could use it.
Writing clean css would also avoid the used of - `overflow-x: hidden;` here

**Comment by Ankita297** [comment]
The horizontal bar was already present there even before i have added the code  so  i just add the code overflow-x:hidden in body So i don't think it is due to footer  . So what should i do in this ?  
WordNook - Google Chrome 09-05-2021 18_06_33

**Comment by Pranav016** [comment]
> The horizontal bar was already present there even before i have added the code so i just add the code overflow-x:hidden in body So i don't think it is due to footer . So what should i do in this ?
> WordNook - Google Chrome 09-05-2021 18_06_33

Ok so just remove the overflow-x and improve the overall css you added because that still has some issues. I'll take care of the horizontal scroll.

File: public/css/styles.css (hunk 1 above)
Comment (Ankita297) [reply public/css/styles.css]: I have remove all the media query  and replace it with % .

**Comment by Ankita297** [comment]
changes done Please check .

**Review by Pranav016 — APPROVED** [review]

**Comment by Ankita297** [comment]
Thank You Sir

**Pull request merged**